molecule/default/playbook.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Prepare
+  hosts: all
+  become: true
+  tasks:
+  - name: Add the inventory into /etc/hosts
+    lineinfile:
+      dest: /etc/hosts
+      regexp: '.*{{ item }}$'
+      line: "{{ hostvars[item]['ansible_enp0s8']['ipv4']['address'] }} {{item}}"
+      state: present
+    when: hostvars[item]['ansible_enp0s8']['ipv4']['address'] is defined
+    with_items:
+      - "{{ groups['all'] }}"
+
+- name: Test
+  hosts: all
+  become: true
+  roles:
+    - role: ansible-role-wireguard
```

Comment:
This didn't worked for me but `githubixx.wireguard` worked.

Reply:
Hum, this is strange.. I think you have a local configuration that is "special".
What is the name of your folder that holds the role?
From what I understand, molecule will add the local folder to the playbook paths (But I;m not 100% sure).

Reply:
These Ansible Galaxy guys really make me angry. They're changing stuff without further notice the second time now... But I guess we were both wrong.
To install the role via Galaxy `ansible-galaxy install githubixx.ansible_role_wireguard` is needed. So I'm pretty sure that you need
```
  roles:
    - role: githubixx.ansible_role_wireguard
```
here. My local directory was called `githubixx.wireguard` and that was the role name at the time I uploaded the role but now it's called `githubixx.ansible_role_wireguard` for whatever reasons. And if I use `- role: githubixx.ansible_role_wireguard` it works for me. And this is the only value that makes sense IMHO because the official role name in Ansible Galaxy is `githubixx.ansible_role_wireguard`.
I assume that you `git clone ...` the role from Github. In that case `- role: ansible-role-wireguard` makes sense but we should stay with the "official" role name in Galaxy.